Slms/Issue 5

garecinos4 opened the pull request:
Publicar microservicio para publications

garecinos4 committed:
Add piblication microservice

garecinos4 committed:
modificar la configuración,  agregar estandares

garecinos4 committed:
Cambiar a estandares

garfenter commented on publication-ms/handlers.js:
```diff
@@ -0,0 +1,104 @@
+        activity: request.payload.activity
+    }
+
+    request.app.db.query('INSERT INTO PUBlICATION (text, state, date, publicator, course_instance, activity) VALUES (:text, :state, :date, :publicator, :course_instance, :activity)', publication, (err, result) => {
```
1) Typo en PUBlICATION, la "L" debería de estar en mayúsculas.

2) Pienso que date no se debería de recibir como parámetro, siempre debería de colocar la fecha del sistema al momento de ser creada.

garfenter replied:
3) Tal vez el valor default del state al crear la publicación debería de ser "ACTIVE"?

garfenter commented on publication-ms/handlers.js:
```diff
@@ -0,0 +1,104 @@
+        state: 'ACTIVE'
+    };
+
+    request.app.db.query('SELECT * FROM PUBlICATION WHERE state = :state', publication, (err, rows, fields) => {
```
@garecinos4 @josedepaz @josuedani @alexredle en este tipo de metodos (findAll) creo que deberíamos de pensar en manejar un standard para la paginación, por ejemplo que el metodo reciba como parametros (La fila inicial y el número de registros a retornar):

Ej.
SELECT \* FROM tbl LIMIT :inicial, :registros;

garecinos4 committed:
1) Typo en PUBlICATION, la "L" debería de estar en mayúsculas.
2) Cambiar para ingresar la fecha del sistema
3) Cambiar estado por default 'ACTIVE' al ingresar
4) Agregar limit y offset

garfenter reviewed (approved):
LGTM

garecinos4 committed:
Agregar limit y offset
